src/components/InputItem.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import { StyleSheet, TextInput as BaseTextInput, View, Text, Animated } from 'react-native';
+
+import { palette, typography, fonts } from 'styles';
+
+interface Props {
+  label: string;
+  suffix?: string;
+  error?: string;
+}
+
+interface State {
+  isActive: boolean;
+  isAnimatedFocused: Animated.Value;
+  value: string;
+}
+
+export class InputItem extends Component<Props, State> {
+  state = {
+    isActive: false,
+    isAnimatedFocused: new Animated.Value(0),
+    value: '',
+  };
+
+  onFocus = () => {
+    this.setState({
+      isActive: true,
+    });
+    // @ts-ignore
```

Comment:
why // @ts-ignore was needed here? It's looking good for me but Typescript was complaining?

Reply:
It said that useNativeDriver is obligatory.